recipes/csvmonkey/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.0.5":
+    url: "https://github.com/dw/csvmonkey/archive/594ad9854b7a77ccb8e26260b1e1a4300ba18465.tar.gz"
```

Comment:
it makes sense to me: https://github.com/dw/csvmonkey/commit/594ad9854b7a77ccb8e26260b1e1a4300ba18465